packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -716,6 +716,65 @@ void main() {
     expect(box.size.width, customWidth);
   });
 
+  testWidgets('The width is determined by the menu entries', (WidgetTester tester) async {
+    const double entryLabelWidth = 100;
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Scaffold(
+          body: DropdownMenu<int>(
+            dropdownMenuEntries: <DropdownMenuEntry<int>>[
+              DropdownMenuEntry<int>(
+                value: 0,
+                label: 'Flutter',
+                labelWidget: SizedBox(width: entryLabelWidth),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    final double width = tester.getSize(find.byType(DropdownMenu<int>)).width;
+    const double menuEntryPadding = 24.0; // See _kDefaultHorizontalPadding.
+    const double leadingWidth = 16.0;
+    const double trailingWidth = 56.0;
+
+    expect(width, entryLabelWidth + leadingWidth + trailingWidth + menuEntryPadding);
+  });
+
+  // Regression test for https://github.com/flutter/flutter/issues/136678.
```

Comment:
I personally think this change is just a new feature to respect the label width. So probably we don't need this comment? but it's just a nit:)